src/app/pages/home/home.component.html

```diff
@@ -57,7 +57,10 @@ <h5 class="card-title text-uppercase text-muted mb-0">Our Volunteers</h5>
               <div class="row">
                 <div class="col">
                   <h5 class="card-title text-uppercase text-muted mb-0">Active Requests</h5>
-                  <span class="h2 font-weight-bold mb-0">{{dashboard.active}}</span>
+                  <div class="row">
+                  <div class="col"><span class="h2 font-weight-bold mb-0">{{dashboard.active}}</span> </div>
+                  <div class="col"><a [routerLink]="['/all-active-requests']" class="btn-sm btn btn-primary">Show all</a></div>
```

Comment:
We don't have the API yet, so you can take out the [routerLink]="['/all-active-requests']" part.